sources/assets/Xenko.Core.Assets/PackageSessionPublicHelper.cs

```diff
@@ -64,8 +69,10 @@ private static void CheckMSBuildToolset()
             // Check that we can create a project
             using (var projectCollection = new Microsoft.Build.Evaluation.ProjectCollection())
             {
-                if (projectCollection.GetToolset("15.0") == null)
-                    throw new InvalidOperationException("Could not find MSBuild toolset 15.0");
+                if (!s_msBuildToolVersions.Select(ver => projectCollection.GetToolset(ver)).Any())
```

Comment:
At this point, to be futureproof, shouldn't we just use `projectCollection.Toolsets` and accept if any of them is higher or equal to 15.0?
Something like `projectCollection.Toolsets.Any(x => new Version(x.ToolsVersion).Major >= 15)` (note: might need some testing to check what ToolsVersion contains exactly)

Reply:
I will look at it tonight. 👍

Reply:
I changed the code and verified that the format is "15.0" and using Version works fine.